lib/worker.js

```diff
@@ -42,7 +42,8 @@ Communication.on('JOB', function(job) {
     configFile = params.configFile
   }
 
-  let modulesPath = find(params.fileDir, 'node_modules')
+  const modulesPath = find(params.fileDir, 'node_modules')
+  const eslintignoreDir = Path.dirname(find(params.fileDir, '.eslintignore'))
```

Comment:
IMO, we should presume that `.eslintrc` and `.eslintignore` are in the same dir, How does the cli `eslint` handle it? Does it search for both?